### PR TITLE
Practice commit

### PR DESCRIPTION
**Pull request by lucasprice**
Some junk that Nick made me commit.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -281,7 +292,7 @@ let drawSoundButton = function() {
   ctx.fillStyle = "black";
   ctx.font = '48px serif';
   if (lightningSound.muted) {
-    ctx.fillText("Sound On", canvas.width - 220, 80);
+    ctx.fillText("Sound OnTop", canvas.width - 220, 80);
```
Comment (nicolasbock) [inline main.js]: You can change that back again :smile:

File: main.js [hunk 2]
```diff
@@ -232,6 +232,16 @@ var obstacleLaser = function(x, y) {
   ctx.fillRect(x + 16, 0, 44, 18);
 };
 
+var obstacleRect = function(x, y) {
+  ctx.rect(500, 500, 100, 25);
```
Comment (nicolasbock) [inline main.js]: Make sure you also use the `x` coordinate. The obstacle is bolted to the floor and that's moving to the left. The `x` coordinate tells the obstacle where the floor is.

Comment (lucasprice) [reply main.js]: I have fixed this (disregard this commit)... see new commit.

File: main.js (hunk 1 above)
Comment (lucasprice) [reply main.js]: Done.

**Pull request merged**